### PR TITLE
Added support for new social icons: YouTube, Tumblr, Gratipay.

### PR DESCRIPTION
**Pull request by nihonjinrxs**
To test, you'll need to add the following to `/node_modules/resume_schema/resume.json` in the `basics.profiles` segment:

```javascript
      {
        "network": "Tumblr",
        "username": "mytumblrusername"
      },
      {
        "network": "Youtube",
        "username": "myyoutubeusername"
      },
      {
        "network": "Gratipay",
        "username": "mygratipayusername"
      }
```

Note that for Gratipay support, I had to bump the Font Awesome dependency up to version 4.3.0.

Happy to discuss.

### CODE REVIEW

**Comment by mudassir0909** [comment]
I guess you might need to rebase

**Comment by nihonjinrxs** [comment]
OK, let's finish the Skype fix first, then I'll rebase and re-push.

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Responsive theme for [JsonResume](https://jsonresume.org/) inspired by card layo
 * facebook
 * angellist
 * bitbucket
+* googleplus
```
Comment (mudassir0909) [inline README.md]: Can you update this with youtube, tumblr & gratipay icons as well ?

**Comment by nihonjinrxs** [comment]
I've merged in current master, fixed the `README.md` file as requested, and resolved merge conflicts.  This one should be ready to go.

**Pull request merged**